tensoevent/tensoevent.go

```diff
@@ -17,9 +17,9 @@
 *
 *******************************************************************************/
 
-// Package helmevent contains data structures describing the event messages that
-// our CI generates for Helm deployments (i.e. "helm install" and "helm upgrade".
-package helmevent
+// Package tensoevent contains data structures describing the event messages that
+// our CI generates for deployments (i.e. "helm install" and "helm upgrade".
+package tensoevent
```

Comment:
This format is more closely tied to our deployment automation than to Tenso, and also from Tenso's perspective this is only one of several input formats, so I don't think having this be a "Tenso event" is accurate naming.
```suggestion
package deployevent
```